-code-checks

Logs: https://cmssdt.cern.ch/SDT/code-checks/cms-sw-PR-40428/33562

ERROR: Build errors found during clang-tidy run.
```
DataFormats/PatCandidates/src/TriggerObjectStandAlone.cc:420:61: error: no member named 'removeSchedulingTokensFromModuleLabel' in namespace 'edm::path_configuration' [clang-diagnostic-error]
          auto const moduleStrip = edm::path_configuration::removeSchedulingTokensFromModuleLabel(module);
                                                            ^
Suppressed 1072 warnings (1071 in non-user code, 1 with check filters).
--
gmake: *** [config/SCRAM/GMake/Makefile.coderules:129: code-checks] Error 2
gmake: *** [There are compilation/build errors. Please see the detail log above.] Error 2
```

--- a/DataFormats/PatCandidates/src/TriggerObjectStandAlone.cc
+++ b/DataFormats/PatCandidates/src/TriggerObjectStandAlone.cc
@@ -9,6 +9,7 @@
 #include "FWCore/Common/interface/TriggerNames.h"
 #include "FWCore/MessageLogger/interface/MessageLogger.h"
 #include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/Utilities/interface/path_configuration.h"
 #include "FWCore/Utilities/interface/thread_safety_macros.h"
 
 using namespace pat;
@@ -416,7 +417,7 @@ std::vector<std::string> const *TriggerObjectStandAlone::allLabels(edm::Paramete
       if (pset->existsAs<vector<string>>(triggerNames.triggerName(i), true)) {
         auto const &modules = pset->getParameter<vector<string>>(triggerNames.triggerName(i));
         for (auto const &module : modules) {
-          auto const moduleStrip = module.front() != '-' and module.front() != '!' ? module : module.substr(1);
+          auto const moduleStrip = edm::path_configuration::removeSchedulingTokensFromModuleLabel(module);
           if (pset->exists(moduleStrip)) {
             const auto &modulePSet = pset->getParameterSet(moduleStrip);
             if (modulePSet.existsAs<bool>("saveTags", true) and modulePSet.getParameter<bool>("saveTags")) {

